doc/source/ray-core/fault_tolerance/ray_components.rst

```diff
@@ -0,0 +1,66 @@
+Failure of Ray components
+=========================
+
+Ray components failure could lead to tasks/objects/actors failure. 
+
+GCS failure
+-----------
+
+GCS failure usually means the Ray cluster failure and everything will be lost.
+To make GCS fault-tolerant, HA storage is needed to be used. Ray support using
+Redis as the backend storage. The user needs to set up the storage first and make
+GCS connected to the Redis.
+
```

Comment:
To make GCS fault-tolerant, HA storage is required. Ray supports Redis as the backend storage for reliability and high availability. You can set up the HA storage first and configure GCS to connect to the Redis.